packages/hint/src/lib/cli/version.ts

```diff
@@ -1,6 +1,6 @@
-import { logger } from '@hint/utils';
-import { loadHintPackage } from '../utils/packages/load-hint-package';
+import { logger, packages } from '@hint/utils';
 
+const { loadHintPackage } = packages;
```

Comment:
Do any of the imports in this file need to change since `../utils/packages/load-hint-package` still re-exports `loadHintPackage`?